api/pyproject.toml

```diff
@@ -4,42 +4,47 @@ version = "1.1.1"
 description = "API Interface for WFP PRISM"
 authors = ["Ovio"]
 license = "Apache 2.0"
+package-mode = false
 
 [tool.poetry.dependencies]
 PyJWT = "2.4.0"
-python = ">=3.10,<3.11"
+python = ">=3.12,<3.13"
 requests = "^2.32.4"
-playwright = "1.38.0"
-fastapi = "^0.109.1"
-schemathesis = "^3.19.7"
+playwright = "1.53.0"
+fastapi = "^0.114.0"
+httpx = "^0.28.0"
 uvicorn = { extras = ["standard"], version = "^0.23.2" }
 sqlalchemy = "^1.4.40"
 pydantic = { version = "1.10.13", extras = ["email"] }
 python-dateutil = "2.8.2"
 types-python-dateutil = "2.8.19"
 psycopg2 = "^2.9.9"
-rasterstats = "0.19.0"
+rasterstats = "0.20.0"
 boto3 = "^1.28.61"
 cachetools = "^5.3.1"
 odc-geo = "^0.4.1"
-odc-stac = "^0.3.11"
-dask = "2024.10.0" # temporary fix for https://github.com/opendatacube/odc-stac/issues/199
+odc-stac = "^0.4.0"
+dask = "^2025.5.0"
```

Comment:
We might not need to set this explicitely anymore based on the comments you removed?